Integrations/integration-DemistoRESTAPI.yml

```diff
@@ -0,0 +1,98 @@
+description: Use Demisto server REST APIs
+configuration:
+- display: Demisto Server URL
+  name: url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Demisto Server API Key
+  name: apikey
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Trust certificate (insecure)
+  name: insecure
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: User system proxy settings
+  name: proxy
+  defaultvalue: "true"
+  type: 8
+  required: false
+script:
+  script: |
+    var serverURL = params.url;
+    if (serverURL.slice(-1) === '/') {
+        serverURL = serverURL.slice(0,-1);
+    }
+
+    var sendRequest = function(method, url, body) {
+        var requestUrl = serverURL;
+        if (url.slice(-1) != '/') {
```

Comment:
Don't you want to get the first char here?

Reply:
you have a bug here. 
And you should support all: `/user/`, `/user` , `user` (the last one is the least important)

Reply:
we do support all 3

Reply:
![image](https://user-images.githubusercontent.com/1042046/33019180-f2137e2c-ce01-11e7-8f5f-85b038589dbb.png)